src/services/MovieService.js

```diff
@@ -1,7 +1,9 @@
 import ApiService from './ApiService';
+import { format } from 'util'
 
 const ENDPOINTS = {
-  MOVIES: '/api/movies/'
+  MOVIES: '/api/movies/',
+  RELATED_MOVIES: '/api/movies/%s/related'
```

Comment:
ne bi isao sa ovim pristupom, mislim ovdje kreiramo samo dodatan endpoint koji ne radi nista vise nego dovuce sve ostale filmove, mozemo iskorisit ovako nesto https://www.django-rest-framework.org/api-guide/filtering/#filtering-and-object-lookups , znaci da ti bukvalno stavis kako query parametre `genre=<value>&exclude__id=1`, ovaj drugi parametar sluzi da bi exclude-ovao taj film za koji trazis zanrove i ovo poprilicno ide lagano na nivou djanga i samim tim izbjegavamo kreiranje nove rute nepotrebno, da smo imali neku vecu provjeru, na osnovu nekih pregleda korisnilovih itd.. onda bi pravili poseban endpoint i radili kroz njega sve, ovako je samo dodatan posao